src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -53,7 +53,7 @@ case class RawlsBillingProject(projectName: RawlsBillingProjectName, users: Set[
 object UserAuth {
 
   def toWorkspaceAccessGroupName(workspaceName: WorkspaceName, accessLevel: WorkspaceAccessLevel) =
-    s"rawls ${workspaceName.namespace}/${workspaceName.name} ${accessLevel}"
+    s"${workspaceName.namespace}/${workspaceName.name} ${accessLevel}".take(64) // group names have a 64 char limit
```

Comment:
Since we're guarding for the pathological case here, I'd go one step further: if we get to 64 chars before the accessLevel, we won't be able to create the 3 groups.  Maybe this?  

`s"${workspaceName.namespace}/${workspaceName.name}".take(57) + " ${accessLevel}"`